src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -2835,6 +2836,31 @@ void statusOfTlrRequestShouldBeChangedIfAssociatedItemCheckedIn() {
     assertThat(requestAssociatedWithFirstItem.getString("status"), is(OPEN_AWAITING_PICKUP));
   }
 
+  @Test
+  void pageRequestShouldNotChangeItemStatusIfFailsWithoutRequestDate() {
+    UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+    var item = itemsFixture.basedUponSmallAngryPlanet();
+    var requester = usersFixture.steve();
+
+    Response response = requestsClient.attemptCreate(new RequestBuilder()
+      .page()
+      .fulfilToHoldShelf()
+      .withRequesterId(requester.getId())
```

Comment:
```suggestion
      .withRequesterId(usersFixture.steve()getId())
```
I'm not sure we need variable requester

Reply:
done